### PR TITLE
Improved the folder structure in IgnoredModels

### PR DESCRIPTION
**Pull request by Whixx**

### CODE REVIEW

**Review by jocke1995 — DISMISSED** [review]

**Review by jakkeez — APPROVED** [review]

**Review by jakkeez — DISMISSED** [review]
Will be approved at a later stage

**Review by LeoWikstrom — DISMISSED** [review]

File: StortSpelProjekt/Engine/src/Renderer/AnimatedModel.h [hunk 1]
```diff
@@ -40,6 +40,8 @@ class AnimatedModel : public Model
     void ResetAnimations();
 
 private:
+    friend class CopyPerFrameTask;
```
Comment (jocke1995) [inline StortSpelProjekt/Engine/src/Renderer/AnimatedModel.h]: Remove this

File: StortSpelProjekt/Engine/src/Renderer/DX12Tasks/CopyPerFrameTask.cpp [hunk 2]
```diff
@@ -41,13 +41,22 @@ void CopyPerFrameTask::Execute()
...
+		volatile std::vector<DirectX::XMFLOAT4X4>* matrices = (reinterpret_cast<std::vector<DirectX::XMFLOAT4X4>*>(const_cast<void*>(std::get<2>(tuple))));
+		volatile PointLight* kebab = (PointLight*)(std::get<2>(tuple));
+
 		copyResource(
 			commandList,
 			std::get<0>(tuple),		// UploadHeap
 			std::get<1>(tuple),		// DefaultHeap
 			std::get<2>(tuple));	// Data
+
+		i++;						//TODO
```
Comment (jocke1995) [inline StortSpelProjekt/Engine/src/Renderer/DX12Tasks/CopyPerFrameTask.cpp:59]: Ok to have this until we find the bug.

**Review by jocke1995 — CHANGES_REQUESTED** [review]
1 thing you forgot to remove

**Review by LeoWikstrom — APPROVED** [review]

**Review by jocke1995 — APPROVED** [review]

**Pull request merged**